app/src/main/res/layout/category_item_view.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/category_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/sf_pro_text"
+        android:gravity="center"
+        android:paddingLeft="@dimen/small_125"
+        android:paddingRight="@dimen/small_125"
+        android:textSize="@dimen/small_135"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="@string/category" />
+
+    <LinearLayout
```

Comment:
Do we need this linear layout here? I think we can simply adjust that underline by using constraint layout attritbutes